x/operator/types/expected_keepers.go

```diff
@@ -94,26 +129,54 @@ func (MockOracle) GetPriceChangeAssets(_ sdk.Context) (map[string]*PriceChange,
 	return nil, nil
 }
 
-type MockAvs struct{}
+func (MockOracle) GetMultipleAssetsPrices(_ sdk.Context, assets map[string]interface{}) (map[string]Price, error) {
+	ret := make(map[string]Price, 0)
+	for assetID := range assets {
+		ret[assetID] = Price{
+			Value:   sdkmath.NewInt(1),
+			Decimal: 0,
+		}
+	}
+	return ret, nil
+}
+
+type MockAvs struct {
+	AssetsKeeper AssetsKeeper
+}
 
-func (MockAvs) GetAvsSupportedAssets(_ sdk.Context, _ string) (map[string]interface{}, error) {
-	// set USDT as the default asset supported by AVS
+func (a MockAvs) GetAvsSupportedAssets(ctx sdk.Context, _ string) (map[string]interface{}, error) {
+	// set all registered assets as the default asset supported by mock AVS
 	ret := make(map[string]interface{})
-	usdtAssetID := "0xdac17f958d2ee523a2206206994597c13d831ec7_0x65"
-	ret[usdtAssetID] = nil
+	allAssets, err := a.AssetsKeeper.GetAllStakingAssetsInfo(ctx)
+	if err != nil {
+		return nil, err
+	}
+	for assetID := range allAssets {
+		ret[assetID] = nil
+	}
 	return ret, nil
 }
 
-func (MockAvs) GetAvsSlashContract(_ sdk.Context, _ string) (string, error) {
+func (a MockAvs) GetAvsSlashContract(_ sdk.Context, _ string) (string, error) {
```

Comment:
can standardize the form. Capitalize all AVS.

Reply:
Yeah, it will be fixed in the subsequent PR.